src/entities/utils/calculateProportionalAmounts.ts

```diff
@@ -0,0 +1,35 @@
+import { Hex, Address, parseUnits } from 'viem';
+import { InputAmount } from '@/types';
+
+export function calculateAddLiquidityProportionalAmounts(
+    pool: {
+        id: Hex;
```

Comment:
I'd make a few adjustments:
1. rename `calculateAddLiquidityProportionalAmounts` with `calculateProportionalAmounts`
2. expect pool `address` instead of `id` so we don't need to manipulate it